addons/breathing/functions/fnc_getETCo2.sqf

```diff
@@ -0,0 +1,61 @@
+#include "..\script_component.hpp"
+/*
+ * Author: apo_tle
+ * Calculate patient's ETCo2 level based on status of their vitals. ETCo2 is not its own vital as it does not need to be,
+ * rather I've implemented it as more like a diagnostic tool.
+ *
+ * Arguments:
+ * 0: Patient <OBJECT>
+ *
+ * Return Value:
+ * ETCo2 level <INT>
+ *
+ * Example:
+ * [player] call kat_breathing_fnc_getETCo2;
+ *
+ * Public: No
+ */
+
+ params ["_patient"];
+
+if !(alive _patient) exitWith {0};
+
+_pr = _patient getVariable [QACEGVAR(medical,heartRate), 0];
```

Comment:
```suggestion
private _pr = _patient getVariable [QACEGVAR(medical,heartRate), 80];
```

Reply:
Also every where else in this file.

Reply:
you can use macros too, i think it should work without issues
```suggestion
private _pr = GET_HEART_RATE(_patient);
```